observability/src/lib.rs

```diff
@@ -79,22 +72,16 @@ where
 
     // Both tracer and meter are configured with environment variables.
     let meter = opentelemetry_otlp::new_pipeline()
-        .metrics(
-            selectors::simple::inexpensive(),
```

Comment:
The default aggregation is now [this](https://docs.rs/opentelemetry_sdk/latest/opentelemetry_sdk/metrics/reader/struct.DefaultAggregationSelector.html), I couldn't find the inexpensive one, seems to have been removed.

Reply:
Seems fine!